packages/discord.js/typings/index.d.ts

```diff
@@ -5411,6 +5411,8 @@ interface GuildAuditLogsTypes {
 export type GuildAuditLogsActionType = GuildAuditLogsTypes[keyof GuildAuditLogsTypes][1] | 'All';
 
 export interface GuildAuditLogsEntryExtraField {
+  [AuditLogEvent.MemberKick]: { integrationType: string };
+  [AuditLogEvent.MemberRoleUpdate]: { integrationType: string };
```

Comment:
These will be `null` if `data.integration_type` is not present, i.e. the kick or role update was not caused by an integration.

```suggestion
  [AuditLogEvent.MemberKick]: { integrationType: string } | null;
  [AuditLogEvent.MemberRoleUpdate]: { integrationType: string } | null;
```